storagemarket/provider.go

```diff
@@ -82,19 +80,19 @@ func NewProvider(repoRoot string, dealsDB *db.DealsDB, fullnodeApi v1api.FullNod
 
 	return &Provider{
 		// TODO: pass max transfer duration as a param
-		config:      Config{MaxTransferDuration: 30 * time.Second},
-		Address:     addr,
-		newDealPS:   newDealPS,
-		fs:          fs,
-		fullnodeApi: fullnodeApi,
```

Comment:
I am surprised we don't need this here, but probably because of all the commented-out code right now.

Reply:
Yep we'll need it when we want to watch deals on chain after they've started sealing